ui/raidboss/data/04-sb/alliance/orbonne_monastery.js

```diff
@@ -313,8 +331,9 @@
     {
       id: 'Orbonne Cid Crush Helm Tank',
       regex: / 15:\y{ObjectId}:The Thunder God:3753:Crush Helm:\y{ObjectId}:(\y{Name}):/,
-      regexDe: / 15:\y{ObjectId}:Cidolfus:3753:Zenitspaltung:\y{ObjectId}:(\y{Name}):/,
+      regexDe: / 15:\y{ObjectId}:Cidolfus:3753:Himmelsbombardement:\y{ObjectId}:(\y{Name}):/,
       regexFr: / 15:\y{ObjectId}:Cid Le Dieu De La Foudre:3753:Bombardement Céleste:\y{ObjectId}:(\y{Name}):/,
+      regexJa: / 15:\y{ObjectId}:雷神シド:3753:星天爆撃打Crush Helm:\y{ObjectId}:(\y{Name}):/,
```

Comment:
This line _feels_ wrong; should it really have `Crush Helm` in English intertwined with the Japanese? It happens again below with `Crush Armor` as well.

Reply:
Thought i solved this bug and replace all the damage it had done. Thanks for pointing this out :) and again: Will be fixed with next commit